cmd/kube-controller-manager/app/core.go

```diff
@@ -323,6 +329,11 @@ func startAttachDetachController(ctx ControllerContext) (http.Handler, bool, err
 		return nil, true, fmt.Errorf("failed to probe volume plugins when starting attach/detach controller: %v", err)
 	}
 
+	filteredDialOptions, err := options.ParseDialHostDenyList(ctx.ComponentConfig.PersistentVolumeBinderController.DialHostCIDRDenylist)
```

Comment:
Is it odd that that the options for the other volume controllers come from the PV controller config? Should we have a separate "VolumeControllers" config that applies to all?

Reply:
That's true, I had set it up this way when it was only being used in the controller manager.

However I confess to not really understanding how these configurations are used and what the implications for how they're organized are.

Reply:
> I had set it up this way when it was only being used in the controller manager.

is this used in other components? I'm not seeing flags or config that would enable that